projects/storefrontapp-e2e-cypress/cypress/integration/regression/site-context/currency/currency-cart-page.e2e-spec.ts

```diff
@@ -1,24 +1,32 @@
-import { manipulateCartQuantity } from '../../../../helpers/cart';
 import * as siteContextSelector from '../../../../helpers/site-context-selector';
+import { switchSiteContext } from '../../../../support/utils/switch-site-context';
 
 describe('Currency switch - cart page', () => {
   const cartPath = siteContextSelector.CART_PATH;
-  const jpCurrency_per_item = ' ¥9,720 ';
-  const jpCurrency_total = ' ¥29,160 ';
+  let cartId = '';
 
   before(() => {
     cy.window().then((win) => win.sessionStorage.clear());
     cy.requireLoggedIn();
-    cy.visit('/');
-    manipulateCartQuantity();
-  });
 
-  siteContextSelector.stub(
-    siteContextSelector.CART_REQUEST,
-    siteContextSelector.CART_REQUEST_ALIAS
-  );
+    cy.window().then((win) => {
+      const savedState = JSON.parse(
+        win.localStorage.getItem('spartacus-local-data')
+      );
+      const accessToken = savedState.auth.userToken.token.access_token;
+      cy.addToCart('300938', '3', accessToken).then((cartCode) => {
+        cartId = cartCode;
+      });
+
+      cy.visit('/cart');
+    });
+  });
 
   describe('cart page', () => {
+    const baseUrl = `${Cypress.env('API_URL')}/${Cypress.env(
+      'OCC_PREFIX'
+    )}/${Cypress.env('BASE_SITE')}`;
+
     it('should change currency in the url', () => {
```

Comment:
need to add cy.route('GET', siteContextSelector.CART_REQUEST).as(
        siteContextSelector.CART_REQUEST_ALIAS
      ); 

as this `it` uses `cart_request_alias`